assets/app/scripts/controllers/overview.js

```diff
@@ -334,6 +355,11 @@ angular.module('openshiftConsole')
       }
     };
 
+    function isCustomHost(route) {
+      var defaultHost = route.serviceName + '-' + route.metadata.namespace;
```

Comment:
might be worth annotating the route with the host generator so clients can know if the host was generated